7_react/react1/week3/vite-project/src/components/TodoList.jsx

```diff
@@ -0,0 +1,65 @@
+import PropTypes from "prop-types";
+import Border from "./Border";
+
+function TodoList({ todos, toggleDone, deleteTodo, editTodo }) {
```

Comment:
This component is quite crowded, but it gets the job done. 💪 Can you think of a way to split the functionality up a bit to make it more digestible? 😄

Reply:
@NVedsted I do not know actually how to do it correct, or you mean separate it into smaller components in the separate files? Like TodoItem, TodoEdit,TodoList smth like that? :)

Reply:
Yes, exactly like that. 😄